controller/space.go

```diff
@@ -309,9 +309,13 @@ func ConvertSpaces(request *goa.RequestData, spaces []*space.Space, additional .
 // ConvertSpace converts between internal and external REST representation
 func ConvertSpace(request *goa.RequestData, p *space.Space, additional ...SpaceConvertFunc) *app.Space {
 	selfURL := rest.AbsoluteURL(request, app.SpaceHref(p.ID))
-	relatedIterationList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/iterations", p.ID.String()))
-	relatedAreaList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/areas", p.ID.String()))
-	relatedCodebasesList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/codebases", p.ID.String()))
+	spaceIDStr := p.ID.String()
+	relatedIterationList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/iterations", spaceIDStr))
+	relatedAreaList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/areas", spaceIDStr))
+	relatedCodebasesList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/codebases", spaceIDStr))
+	relatedWorkItemList := rest.AbsoluteURL(request, app.SpaceWorkItemsHref(spaceIDStr))
+	relatedWorkItemTypeList := rest.AbsoluteURL(request, app.SpaceWorkItemTypesHref(spaceIDStr))
+	relatedWorkItemLinkTypeList := rest.AbsoluteURL(request, app.SpaceWorkItemLinkTypessHref(spaceIDStr))
```

Comment:
fixed in https://github.com/almighty/almighty-core/pull/1033/commits/7bcb70ce3afbc6b62c1425bf70236abc699cccfc